app/client/components/accountoverview/accountOverviewCancelledCard.tsx

```diff
@@ -3,7 +3,7 @@ import { space } from "@guardian/src-foundations";
 import { brand, brandAlt, neutral } from "@guardian/src-foundations/palette";
 import { textSans } from "@guardian/src-foundations/typography";
 import React from "react";
-import { formatDateStr } from "../../../shared/dates";
+import { parseDate } from "../../../shared/dates";
```

Comment:
It would have saved a lot of updated lines if we had retained `formatDateStr` and had it wrap `parseDate`? It doesn't require any additional info. Is this simply continuing a change already made on other files in the platform?

Reply:
Hey, that's an interesting point. The parseDate function does a bit more than the original formatDateStr function which just takes a date string input and spits out a date string output ... the parseDate function takes a date string input (or nothing if you want now) and spits out an object with a date object, a date string (which can be in any custom output format) and various helper functions. 

I guess where we are only interested in imputing a date string and outputting a date string and there isn't any subsequent date manipulation then I could recreate the formatDateStr function in dates.ts and as you said have it wrap the parseDate function, something like this:

```
export const formatDateStr = (inputDateStr: string, outputDateFormat:string) => {
  return parseDate(inputDateStr).dateStr(outputDateFormat);
}
```

An assumption is then made that the input date string format is always `"yyyy-MM-dd"; // example: 1969-07-16`

If that isn't a safe assumption then I would have to change the shape of the formatDateStr function and change everywhere that uses it.